apps/www/components/Article/useSchema.tsx

```diff
@@ -137,8 +137,8 @@ const useSchema = ({
       getSchemaCreator(template)({
         t,
         Link: HrefLink,
-        plattformUnauthorizedZoneText: inNativeIOSApp
-          ? t('plattformUnauthorizedZoneText/ios')
+        plattformUnauthorizedZoneText: inNativeApp
```

Comment:
OMG what is this 😂

Reply:
i discovered this yesterday too… apparently it's not the kachels/teaser. maybe we had sales ones at some point?? your guess it as good as mine.